next/components/ui/Breadcrumbs/DesktopBreadcrumbs.tsx

```diff
@@ -21,9 +26,13 @@ const DesktopBreadcrumbs = ({ breadcrumbs }: BreadcrumbsProps) => {
           <li className="text-small flex gap-1 font-medium" key={index}>
             <BreadcrumbsChevronIcon className="shrink-0" />
             {breadcrumb.path && !isLast ? (
-              <MLink href={breadcrumb.path} variant="underlined">
-                {breadcrumb.title}
-              </MLink>
```

Comment:
But TBH, I don't see a reason to use a Button here, MLink is completely okay, isn't it?

Reply:
Well It would be sufficient. But since Button contains and uses MLink when handling href. Isn't it ok to handle these links trough Button with href?

Reply:
No, if it would change the design even slightly, please use MLink.